scripts/functions/similarity_metric.py

```diff
@@ -0,0 +1,662 @@
+'''
+Author: Alexandra Lee
+Date Created: 30 August 2019
+
+Scripts to generate simulated data and visualize it
+'''
+
+import os
+import sys
+import ast
+import pandas as pd
+import numpy as np
+import random
+import glob
+import warnings
+warnings.filterwarnings(action='ignore')
+
+sys.path.append("../")
+from functions import cca_core
+
+from sklearn.decomposition import PCA
+from numpy.random import seed
+randomState = 123
+seed(randomState)
+
+
+def sim_svcca(simulated_data_file,
+              permuted_simulated_data_file,
+              num_experiments,
+              use_pca,
+              num_PCs,
+              base_dir,
+              analysis_name):
+    '''
+    We want to determine if the different batch simulated data is able to capture the
+    biological signal that is present in the original data:
+    How much of the real input data is captured in the simulated batch data?
+
+    In other words, we want to compare the representation of the real input data and the simulated batch data.
+    We will use **SVCCA** to compare these two representations.
```

Comment:
there are two modes: pca +cca and svcca. this statement + reference only applies to one of two modes.

Reply:
I have added a note to the description.